### PR TITLE
Feature/work

### PR DESCRIPTION
**Pull request by ayrtx**

### CODE REVIEW

File: app/templates/works/base.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% load staticfiles %}
```
Comment (tadityar) [inline app/templates/works/base.html]: templates should inherit from the templates/base.html instead, move important works related stuff to the "children" template

File: app/templates/works/work_list.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% extends 'works/base.html' %}
```
Comment (tadityar) [inline app/templates/works/work_list.html]: See the comment about inheriting from templates/base.html

File: app/views.py [hunk 3]
```diff
@@ -93,6 +93,77 @@ def composer_remove(request, pk):
     composer = SqlMethods.delete_composer(pk)
     return redirect('composer_list')
 
+def work_list(request, **kwargs):
```
Comment (tadityar) [inline app/views.py]: For the views, there are more generalized methods available at the current `origin/master`. It should be in `utils.py`

Please do the following:
1. `git checkout master`
2. `git pull`
3. `git checkout feature/work`
4. `git rebase master`
5. Solve whatever conflict you have
6. Commit the conflict resolution
7. Change the code in this file to use the more generalised method.
8. Commit code changes
9. `git push -f feature/work` <- a little bit dangerous, make sure you push force to `feature/work` branch and not master

File: app/templates/works/base.html (hunk 1 above)
Comment (johsi-k) [reply app/templates/works/base.html]: ogay 🌮

File: app/templates/works/work_list.html (hunk 2 above)
Comment (johsi-k) [reply app/templates/works/work_list.html]: removed 🚌

**Pull request merged**